packages/datetime/src/dateInput.tsx

```diff
@@ -320,4 +321,36 @@ export class DateInput extends AbstractComponent<IDateInputProps, IDateInputStat
     private setInputRef = (el: HTMLElement) => {
         this.inputRef = el;
     }
+
+    /**
+     * Translate a moment into a Date object, adjusting the moment timezone into the local one.
+     * This is a no-op unless moment-timezone's setDefault has been called.
+     */
+    private fromMomentToDate = (date: moment.Moment) => {
```

Comment:
a different name would be helpful to clarify that this is not a `Date`. `momentDate`?